src/views/employeeDetails.hbs

```diff
@@ -237,6 +234,10 @@
             <div><input value="{{obj.exemptionUniversity}}" type="text" name="exemptionUniversity" class="input-field input-field-div4"
                 readonly></div>
           </li>
+          <li>
+            <span>اعفاء المواصلات</span>
+            <div><span>{{objExemption.transportationsExemptions}}</span></div>
```

Comment:
Please explain what is `transportationsExemptions`?

Reply:
translate it